samples/app/src/main/kotlin/com/bumble/appyx/app/node/slideshow/WhatsAppyxSlideShow.kt

```diff
@@ -64,14 +65,19 @@ import kotlinx.parcelize.Parcelize
 class WhatsAppyxSlideShow(
     buildContext: BuildContext,
     autoAdvanceDelayMs: Long? = null,
+    private val isInPreviewMode: Boolean = false,
```

Comment:
These two params belong together. If we're in preview, we also have an `autoAdvanceDelayMs`. Maybe package it into a `LaunchMode` sealed class. (Separate PR ok too to unblock this as a quick fix).